src/service/resources-impl.js

```diff
@@ -53,6 +57,13 @@ const FOCUS_HISTORY_TIMEOUT_ = 1000 * 60; // 1min
 const FOUR_FRAME_DELAY_ = 70;
 const MAX_BUILD_CHUNK_SIZE = 10;
 
+/** @const {!{id: string, control: string, experiment: string}} */
+export const INTERSECT_RESOURCES_EXP = {
```

Comment:
Any chance of putting this into a separate file? I've been much more preferring a separate declaration file lately. We can call it `intersec-resources-exp.js` or such.

Reply:
done.